ci/kind/kind-setup.sh

```diff
@@ -78,6 +84,22 @@ function get_encap_mode {
   echo "--encap-mode $ENCAP_MODE"
 }
 
+function add_option {
+     options+=(""$1" "$2"")
+}
+
+function check_action {
```

Comment:
not sure declaring a function for this helps with readability

Reply:
I thought it would increase readability ,should I remove it ?

Reply:
yes, I think you should delete the function and move the code underneath the `while` loop